packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -6,7 +6,7 @@ import {
   isValidDateTime,
   unixTimestampToMoment,
   formatUnixTimestamp,
-} from './timestampUtilities';
+} from './Utilities/timestampUtilities';
```

Comment:
Usually, the utility folders are lowercase, do you mind making this change?

Reply:
I have updated the imports and renamed the folder, but for some reason GitHub doesn't register the change from `Utilities` to `utilities` and it did not update the name even though locally everything is correct. Currently looking into it.

Reply:
This is done now :) Git doesn't like case sensitivity